Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -432,6 +432,27 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.17,
 )
 
+#Workflow to enable displacedRegionalStep tracking iteration
+class UpgradeWorkflow_displacedRegional(UpgradeWorkflowTracking):
+    def setup__(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+    def condition_(self, fragment, stepList, key, hasHarvest):
+        return '2021' in key
+upgradeWFs['displacedRegional'] = UpgradeWorkflow_displacedRegional(
+    steps = [
+        'Reco',
+        'RecoFakeHLT',
+        'RecoGlobal',
+        'RecoNano',
+    ],
+    PU = [],
+    suffix = '_displacedRegional',
+    offset = 0.71,
```

Comment:
the offset selection is still nebulous to me. I usually chose something not used already that's `+1` from the last similar workflow

Reply:
@srimanob do you have suggestions?

Reply:
I think `.71` or `.701` is OK. If you will have more tracking workflows, I would suggest .701, because .71 can have only 3 more, before hits .75 (HLT Phase-2).

Reply:
OK, I changed it to .701.